governator-test/src/main/java/com/netflix/governator/guice/test/AnnotationBasedTestInjectorManager.java

```diff
@@ -53,29 +54,30 @@
     private final TestPropertyOverrideAnnotationReader testPropertyOverrideAnnotationReader = new TestPropertyOverrideAnnotationReader();
     private TestCompositeConfig testCompositeConfig;
 
-    public AnnotationBasedTestInjectorManager(Class<?> classUnderTest) {
+    public AnnotationBasedTestInjectorManager(Class<?> classUnderTest, Class<? extends MockHandler> defaultMockHandlerClass) {
```

Comment:
Only breaking for people who've extended our test runner. Not breaking for people writing tests. And it does not require Spock. I added this type to let Spock users choose whether or not they wanted Spock mock handling or Mockito (as each offers a different set of pros and cons). It also serves as an extension point for introducing other Mock frameworks or customizing the behavior of existing ones.